### PR TITLE
[TW-1110] Autosave api client

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Added new Autosave option in settings
* The feature is in beta, and it's only available in Free, Basic, and Professional teams
* Added brief mentions about the feature in a few places

[Beta site](https://tw-1110-autosave-api-client.learning.postman-beta.com/docs/getting-started/installation/settings/#application)

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Review by saialekhya-001 — COMMENTED** [review]
@MatthewPinsker-postman  can you share the preview of the document if available?

File: src/pages/docs/getting-started/settings.md [hunk 1]
```diff
@@ -100,6 +100,10 @@ When you send a form-data or binary file with a request body, Postman saves a pa
 
 ### Application
 
+* **Autosave** - Turn this on to automatically save your changes to collections, requests, and environments. If you've added an integration that [backs up a collection](https://go.postman.co/integrations/browse?category=backup), turn this on to automatically back up your changes to your collection. Turn this off to manually save your changes.
+
```
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/settings.md:104]: This is now updated. Thanks!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/getting-started/installation/settings.md [hunk 2]
```diff
@@ -100,7 +100,10 @@ When you send a form-data or binary file with a request body, Postman saves a pa
 
 ### Application
 
-<!-- * **Autosave** - Turn this on to automatically save any changes you make to requests and collections. -->
+* **Autosave** - Turn this on to automatically save your changes to collections, requests, and environments. If you've added an integration that [backs up a collection](https://go.postman.co/integrations/browse?category=backup), turn this on to automatically back up your changes to your collection when changes are autosaved. Turn this off to manually save your changes.
```
Comment (giridharvc7) [inline src/pages/docs/getting-started/installation/settings.md:103]: This might not be accurate, we only save changes to Postman cloud at this point.

Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/installation/settings.md:103]: I'll follow up with @saialekhya-001. It's my understanding that when users manually save changes to their collection, the changes automatically commit to their repo (if a collection backup integration is added). If this is not the default behavior when autosave is turned on, then we should call this out.

Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/installation/settings.md:103]: @shubhbhargav confirmed that the sentence about collection backup integrations is OK.

**Pull request merged**